test/int_test.bats

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+@test "test disk threshold value to be equal to default value" {
+    [[ $(awk -F "[><]" '/available_disk_threshold/{print $3}' packaging/iofog-agent/etc/iofog-agent/config_new.xml) = $(sudo docker exec iofog-agent iofog-agent info | grep "Available Disk Threshold" | awk '{print $5}') ]]
```

Comment:
Just 'cos you're in bash, doesn't mean you have to right crazy one line code! ;)

Reply:
Will split the code to make it more understandable